R/reportEmi.R

```diff
@@ -496,19 +494,20 @@ reportEmi <- function(gdx, output = NULL, regionSubsetList = NULL, t = c(seq(200
                setNames((dimSums(EmiFeCarrier, dim = 3)
                          # subtract industry CCS
                          - dimSums(vm_emiIndCCS[, , emiInd37_fuel], dim = 3)*p_share_CCS
-                         # subtract synthetic and biogenic carbon contained in industrial feedstocks
+                         # subtract synthetic and biogenic carbon contained in chemical feedstocks that don't return to the atmosphere (e.g. non-incinerated plastics)
                          - dimSums(plastic_CDR, dim=3)
                         )*GtC_2_MtCO2,
                         "Emi|CO2|Energy|+|Demand (Mt CO2/yr)")
 
   )
 
  # CO2 emissions from the end-of-life of carbon-bearing products
+ # fixme: wild guess: mybe I have to discount the carbon of biogenic and atmospheric origin from here as well. Nope, wasn't that
 if (!is.null(vm_plasticsCarbon)) {
     out <- mbind(out,
-               setNames(dimSums(vm_feedstockEmiUnknownFate, dim=3)* GtC_2_MtCO2,
+               setNames((dimSums(vm_feedstockEmiUnknownFate, dim=3))* GtC_2_MtCO2,
                         "Emi|CO2|Energy|Waste|+|Feedstocks unknown fate (Mt CO2/yr)"),
-               setNames(dimSums(vm_incinerationEmi, dim=3)* GtC_2_MtCO2,
+               setNames((dimSums(vm_incinerationEmi, dim=3))* GtC_2_MtCO2,
```

Comment:
What is up with those brackets?

Reply:
ups, I was trying something and forgot to delete those. Now I did.